App/app.py

```diff
@@ -7,27 +8,36 @@
 import transformers as ppb
 
 app = Flask(__name__)
+CORS(app)
 
 
 @app.route("/")
-def bert_knn_recommendation():
-    knn_model = pickle.load(open('Models/BERT-KNN.pkl', 'rb'))
+def content_based_recommendation():
+    return render_template('content_recommendation.html')
+
 
+@app.route("/recommend/<query>", methods=['GET'])
```

Comment:
@ZebraAlgebra you need to add a route similar to what I did. This route will handle your CF matrix inference. You need to give it a different name e.g. `/recommend_user/<query>`